pkg/activitypub/vocab/objectproperty.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vocab
+
+import (
+	"encoding/json"
+	"net/url"
+)
+
+// ObjectProperty defines an 'object' property. The property may be a simple IRI or
```

Comment:
IRI or URI?

Reply:
In Activity Streams they use IRI: https://www.w3.org/TR/activitystreams-core/#urls. Also go-fed called it IRI everywhere so I used the same convention.